### DISCUSSION TITLE
Alternative blocky+postgresql dashboard

### DISCUSSION THREAD

**Discussion by shizunge** [0xERR0R/blocky#860] [General]
I created a grafana dashboard for blocky+postgresql, before I found there is already one.

* Grafana link: https://grafana.com/grafana/dashboards/17996
* Github link: https://github.com/shizunge/blocky-postgresql

Hope you find it useful.
Cheers.

Screenshot0
![screenshot0](https://raw.githubusercontent.com/shizunge/blocky-postgresql/main/dashboard/screenshots/screenshot0.png)

Screenshot1
![screenshot1](https://raw.githubusercontent.com/shizunge/blocky-postgresql/main/dashboard/screenshots/screenshot1.png)

Screenshot2
![screenshot2](https://raw.githubusercontent.com/shizunge/blocky-postgresql/main/dashboard/screenshots/screenshot2.png)

**Comment by kwitsch** [comment]
Looks nice, should be added as external alternative to the documentation.

**Comment by johnmmcgee** [comment]
oh nice.  i like this.
